modules/nw-exposing-service.adoc

```diff
@@ -25,67 +25,54 @@ $ oc project myproject
 ----
 
 ifndef::nodeport[]
-. Run the following command to expose the route:
-+
-[source,terminal]
-----
-$ oc expose service <service_name>
-----
-+
-For example:
+. Run the `oc expose service` command to expose the route:
 +
+
 [source,terminal]
 ----
-$ oc expose service mysql-80-rhel7
+$ oc expose service nodejs-ex
 ----
 +
 .Example output
 [source,terminal]
 ----
-route "mysql-80-rhel7" exposed
+route.route.openshift.io/nodejs-ex exposed
 ----
-
-. Use a tool, such as cURL, to make sure you can reach the service using the
-cluster IP address for the service:
 +
-[source,terminal]
-----
-$ curl <pod_ip>:<port>
-----
+. To verify that the service is exposed, you can use a tool, such as cURL, to make sure the service is accessible from outside the cluster.
 +
-For example:
+.. Use the `oc get route` command to find the route's host name.
 +
 [source,terminal]
 ----
-$ curl 172.30.131.89:3306
+$ oc get route
 ----
 +
-The examples in this section uses a MySQL service, which requires a client
-application. If you get a string of characters with the `Got packets out of order`
-message, you are connected to the service.
-+
-If you have a MySQL client, log in with the standard CLI command:
-+
+.Example output
 [source,terminal]
 ----
-$ mysql -h 172.30.131.89 -u admin -p
+NAME        HOST/PORT                        PATH   SERVICES    PORT       TERMINATION   WILDCARD
+nodejs-ex   nodejs-ex-myproject.example.com         nodejs-ex   8080-tcp                 None
 ----
 +
-.Example output
+.. Use cURL to check that the host responds to a GET request.
++
 [source,terminal]
 ----
-Enter password:
-Welcome to the MariaDB monitor.  Commands end with ; or \g.
-
-MySQL [(none)]>
+$ curl nodejs-ex-myproject.example.com
```

Comment:
or use the curl command with `-I` option, see
```
$ curl http://nodejs-ex-myproject.example.com -I
HTTP/1.1 200 OK
...
```
The response status code 200 means the application is accessible from outside the cluster.

Reply:
That's a good suggestion -- thanks! I've made the change.